analyses/cms-open-data-ttbar/statistical.py

```diff
@@ -0,0 +1,471 @@
+import ROOT
+import os
+from statistical_interference.agc_sample import AGC_Sample
+from statistical_interference.rebinning_tool import RebinningTool
+from statistical_interference.drawer import Visualization, DrawModel
+
+
+def fit_histograms(filename=""):
+    print(f"Starting fitting process using input file: {filename}")
+
```

Comment:
```suggestion
    print(f"Starting fitting process using input file: {filename}")

    ROOT.RooMsgService.instance().setGlobalKillBelow(ROOT.RooFit.WARNING)

```
I would kill any output that is below the "warning" level.